dashboard/app/models/pd/workshop.rb

```diff
@@ -81,6 +81,7 @@ class Pd::Workshop < ActiveRecord::Base
       SUBJECT_CS_IN_S_PHASE_3_SEMESTER_2 = 'Phase 3 - Semester 2'
     ],
     COURSE_CSP => [
+      SUBJECT_CSP_SUMMER_WORKSHOP = 'Summer Workshop',
```

Comment:
Ah ok, I didn't realize the summer workshops happened after workshops 1-4. Let me know if you want me to change it, otherwise I'll wait until I hear from the PLC team.